Services/DataBaseService/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DataBaseService.DbModels;
+using DataBaseService.Interfaces;
+using DTO;
+
+namespace DataBaseService.Repositories
+{
+    public class UserRepository : IRepository<UserEmailPassword>
```

Comment:
Will we create UserInformationRepository for full information of user? How we will distinguish one user from another?

Reply:
Rename to UserCredentialRepository